pkg/ccl/backupccl/restore_data_processor.go

```diff
@@ -87,6 +95,16 @@ const restoreDataProcName = "restoreDataProcessor"
 
 const maxConcurrentRestoreWorkers = 32
 
+// sstReaderOverheadBytesPerFile and sstReaderEncryptedOverheadBytesPerFile were obtained
+// benchmarking external SST iterators on GCP and AWS and selecting the highest
+// observed memory per file.
+const sstReaderOverheadBytesPerFile = 5 << 20
```

Comment:
5MiB?! ooof, that's bigger than some of the actual files

Reply:
Yeah I was surprised as well.